src/utils/jiti.ts

```diff
@@ -0,0 +1,8 @@
+import { type JitiOptions, createJiti } from 'jiti';
+
+const options: JitiOptions = {};
+
+const createLoader = (options: JitiOptions) =>
+  createJiti(process.cwd(), options);
+
+export const jiti = createLoader(options);
```

Comment:
why not just this?

```suggestion
export const jiti = createJiti(import.meta.url);
```

IMO we could even inline this one-liner into `src/runner.ts`, as it is the only file where it is used.

Reply:
I actually explored this during testing, everything seems to work fine, but the key difference is that `import.meta.url` gives the current module path, whereas `process.cwd()` points to the process's root. Using relative imports can break with the former, since it resolves from the module itself (e.g., `jiti.ts` or`node_modules/node-pg-migrate/dist/bundled/index.js`?) so `./migrations` would mean `<jiti.ts-directory>./migrations/`

I saw some older issues about relative paths breaking, so right now `node-pg-migrate` will use always absolute paths:
https://github.com/salsita/node-pg-migrate/blob/f4b4ab4d8714f6bdffbe77effd45338e07d5d306/src/runner.ts#L176-L180

So, using `import.meta.url` should be fine as long as we stick to absolute paths, for reference some tools like Nitro and Nuxt re-export `rootDir` (defaulting to `cwd`), since they might use relative paths or something

https://github.com/nitrojs/nitro/blob/eff61ce8f28c3ebb40605d0da39fc47d53a5992b/src/module.ts#L33
https://github.com/nuxt/nuxt/blob/97832ab24e707b052f9502dc369c00a1be2266bc/packages/kit/src/module/install.ts#L101

> IMO we could even inline this one-liner into `src/runner.ts`, as it is the only file where it is used.

Yeah, moving it to `runner.ts` sounds good. I originally split it out thinking I’d need to re-export or tweak some `Option`s, but so far the only one that seems worth exposing is `debug`, mostly because it helps see how migration files are being resolved. but I’m not sure it’s actually necessary.

Reply:
Moved it to `runner.ts` and also removed the return mock from tests since it wasn't actually testing the files.